app/Services/UserService.php

```diff
@@ -47,4 +48,13 @@ public function updateUser(array $data, User $user) {
         $user->fill($data);
         $user->save();
     }
+    public function getUserEmails()
+    {
+        return $collection = DB::table('users')->pluck('email')->toArray();
+    }
+
+    public function getUserData(array $data, User $user)
+    {
+        return $user->toArray();
```

Comment:
you created UserResource class and didn't use it.
getUser method must return $user
in the controller before response we must use UserResource class to convert user data